addons/binding/org.openhab.binding.mihome/.gitignore

```diff
@@ -0,0 +1,5 @@
+.classpath
```

Comment:
Please remove binding-specific `.gitignore`
(I've submitted my own binding and received same change request :))